src/App.tsx

```diff
@@ -1,11 +1,23 @@
+import ContactDetails from './components/ContactDetails';
+import ContactList from './components/ContactList';
+import CreateContactForm from './components/CreateContactForm';
 import Home from './components/Home'
+import { Routes, Route} from "react-router-dom";
```

Comment:
nit: spacing

Reply:
I'm actually not seeing how this is different from original.